### ISSUE TITLE
[a11y]: The ARIA attributes "aria-label" are not valid for the element <label> with ARIA role "none"

### ISSUE THREAD

**Issue by xubinnb** [carbon-design-system/carbon#15549]
### Package

carbon-components-react

### Browser

Chrome

### Operating System

MacOS

### Package version

v7.59.16

### React version

v17.0.2

### Automated testing tool and ruleset

IBM Equal Access Accessibility Checker - Latest Deployment

### Description

===== ENVIRONMENT =====
Browser: Version 105.0.5195.125 (Official Build) (arm64)
Automatically update Chrome for all users Learn more
Test Tool Type: IBM Accessbility Checker

===== DEFECT DESCRIPTION ===== 
Problem 1: The ARIA attributes "aria-label" are not valid for the element <label> with ARIA role "none"

=====STEPS TO REPRODUCE =====
Go to https://dev.console.test.cloud.ibm.com/projects
Click 'Create'
Input the name and description to create a new project
Go to 'Management' tab
Go to 'Environment' tab
Click 'Add'

Use IBM Equal Access Accessibility Checker to scan

===== EXPECTED RESULT =====
[IBM 4.1.2 Name, Role, Value](https://www.ibm.com/able/requirements/requirements/#4_1_2)
For all user interface components (including but not limited to: form elements, links, and components generated by scripts), the name and role can be programmatically determined; states, properties, and values that can be set by the user can be programmatically set; and notification of changes to these items is available to user agents, including assistive technologies.

===== SCREEN CAPTURE =====
<img width="1281" alt="image" src="https://media.github.ibm.com/user/42629/files/ba9b8418-bc4d-4774-8b34-1b88f7074857">

### Reproduction/example

NA

### Steps to reproduce

=====STEPS TO REPRODUCE =====
Go to https://dev.console.test.cloud.ibm.com/projects
Click 'Create'
Input the name and description to create a new project
Go to 'Management' tab
Go to 'Environment' tab
Click 'Add'

Use IBM Equal Access Accessibility Checker to scan

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by tw15egan** [comment]
Can you please provide a simplified reproduction of the a11y violation via [Stackblitz](https://stackblitz.com/github/carbon-design-system/carbon/tree/main/examples/vite) so we can take a deeper look?

**Comment by xubinnb** [comment]
As @flannanl comment, it was fixed in his own code, so close this issue.
